src/content/docs/backend/API/unit-group_sets.md

```diff
@@ -0,0 +1,360 @@
+| Username     | Username             | header         | string    | Yes       |
+| Auth_Token   | Authentication token | header         | string    | Yes       |
+| unit_id      | The unit id          | path           | integer   | Yes       |
+| group_set_id | Id of the group set  | path           | integer   | Yes       |
+| file         | CSV file upload      | body           | file      | Yes       |
+
+- Response: `200 OK`
+
+- Example Request:
+
+  ```bash
+   curl -X POST \--header 'Accept: application/json' \ --header 'Username: aadmin' \ --header 'Auth_Token: Say13yA497H8wSFFXZxh' \--form 'file=@/path/to/file.csv' \'http://localhost:3000/api/units/1/group_sets/2/groups/student_csv'
+  ```
+
+- Response Body:
+
+```json
+{
+  "result": "Success"
+}
+```
+---
+
+### GET: Download a CSV of groups within a group set
+
+GET /api/units/{unit_id}/group_sets/{group_set_id}/groups/csv
+
+- URL: `/api/units/{unit_id}/group_sets/{group_set_id}/groups/csv`
+- Method: `GET`
+- Parameter:
+
+| Parameter    | Description          | Parameter Type | Data Type | Mandatory |
+| ------------ | -------------------- | -------------- | --------- | --------- |
+| Username     | Username             | header         | string    | Yes       |
+| Auth_Token   | Authentication token | header         | string    | Yes       |
+| unit_id      | The unit id          | path           | integer   | Yes       |
+| group_set_id | Id of the group set  | path           | integer   | Yes       |
+
+- Response: `200 OK`
+
+- Example Request:
+
+  ```bash
+   curl -X GET \--header 'Accept: application/octet-stream' \--header 'Username: aadmin' \--header 'Auth_Token: Say13yA497H8wSFFXZxh' \'http://localhost:3000/api/units/1/group_sets/2/groups/csv'
+  ```
+
+- Response Body:
+  ```
+  (Binary CSV file download)
+  ````
+---
+### POST: Upload a CSV for groups within a group set
+
+POST /api/units/{unit_id}/group_sets/{group_set_id}/groups/csv
+
+- URL: `/api/units/{unit_id}/group_sets/{group_set_id}/groups/csv`
+- Method: `POST`
+- Parameter:
+
+| Parameter    | Description          | Parameter Type | Data Type | Mandatory |
+| ------------ | -------------------- | -------------- | --------- | --------- |
+| Username     | Username             | header         | string    | Yes       |
+| Auth_Token   | Authentication token | header         | string    | Yes       |
+| unit_id      | The unit id          | path           | integer   | Yes       |
+| group_set_id | Id of the group set  | path           | integer   | Yes       |
+| file         | CSV file upload      | body           | file      | Yes       |
+
+- Response: `200 OK`
+
+- Example Request:
+
+  ```bash
+   curl -X POST \--header 'Accept: application/json' \--header 'Username: aadmin' \--header 'Auth_Token: Say13yA497H8wSFFXZxh' \--form 'file=@/path/to/file.csv' \'http://localhost:3000/api/units/1/group_sets/2/groups/csv'
+```
+- Response Body:
+
```

Comment:
The formatting here is broken on the site
![image](https://github.com/user-attachments/assets/85a8c867-c810-4ea9-ad83-f794bce1cbfc)

Reply:
Hi Jake, This is now updated.